examples/api/lib/material/dialog/adaptive_alert_dialog.0.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for [AlertDialog].
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
```

Comment:
Ah just one more minor comment(Sorry!). I'm just thinking to follow Greg's this PR: https://github.com/flutter/flutter/pull/124080, probably it's better to rename the `MyApp` class to `AdaptiveAlertDialogApp` or something else:)